src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_buffer_fusing.cpp

```diff
@@ -450,6 +448,218 @@ static void propagate_padding_to_opt_out_users(program_node& node, cldnn::paddin
     }
 }
 
+bool crop_in_place_optimization::match(crop_node& node) {
+    auto pred_param = node.get_dependency(0).get_kernel_impl_params();
+    auto pred_layout = pred_param->get_output_layout();
+    return (match(node, *node.get_kernel_impl_params(), pred_layout));
+}
+
+bool crop_in_place_optimization::match(const program_node& node,
+                                       kernel_impl_params crop_params,
```

Comment:
to reduce copy at runtime, lets pass argument as refernece (kernel_impl_params&)

Reply:
updated